### DISCUSSION TITLE
Calendar Freezing Thunderbird... on Windows 10 Pro only (not Windows 10 Home)

### DISCUSSION THREAD

**Discussion by Tchibeck** [kewisch/gdata-provider#475] [Support]
Hello world,

I have 3 PC under Windows 10
One under Windows 10 Pro with 64 GB RAM, 2 others, under Windows 10 Home with 16 GB RAM

- My Wife computer, on wich she wants the minimum stricly necessary. On her computer, Thunderbird latest release 102.1.1 works perfectly well with Calendar (Gdata Plug-in)

- My Main computer wich run Windows 10 Pro

- My Spare computer wich run Windows 10 Home, ,and is the most possible copy of my main computer. Exactly same softwares are installed (and on wich data were synch via Dropbox)

(haven't tried under Linux PCs, beeing bored to spend too much time in front of PCs, trying to unfreeze softwares).

My first remark is, off topic, not only Thunderbird is slow to start when Google Data Provider plug-in is active.

Also some other softwares, (stupid i am), as i was used to work directly from a drive in wich all my data were... synchronized with Dropbox.

So, my Directory containing the local files of Thunderbiord is on Dropbox.

I have exported all my local e-mails of them in EML format, out of Dropbox. (10 years of mails.... wich were heavy to deal, asa long as to many files synched on Dropbox make them reindex slower)

And archived out of synch a lot of other data with no link with Thunderbird.

So, now what is my reporting  :

1° Even before extracting 70% of my Dropbox data from synch, both of the PCs working on Windows 10 Home had no problem at all with Gdata Provider (i share my calendar with my wife's computer)

No freezing at all, nore on my spare PC (Windows 10 Home)

Now, i have emptied most of the synch data from Dropbox, uninstalled Thnderbird and all traces of it, reinstall a clean Thunderbird on may main computer runninig Windows 10 Pro.... Thunderbird started to freeze again as soone as i installed Gdata Provider plug-in. Freezing approximatly 25 sesond before working nice.

None of the two other computers freeze : the strat Thunderbird immediatly and run Gdata Provider perfectly.

On My W10 Pro PC, i did resynch each Calendar one by one.

But nothing changed : Thunderbird freeze almost 30 seconds before runnig.

It doesn't freeze in safe mode nore if if deactivate temporarily Gdata Provider plug-in.

So my question is : Do other people experiences this problem on Windows 10 PRO, and could it be possible taht this problem with Gdata Provider occured becaus the OS is W10 PRO ?

Should i ask Microsoft to downgrade my license to W10 Home, knowing that i bought this PC without the choice : it was pre-installed with W10 PRO, wich i have no use ?

Sorry for my poor english, and thanks having read me.

**Comment by wsmwk** [comment]
Windows 10 pro versus home should make no difference.

Having data in dropbox isn't strictly supported and is not recommended.

Just to clarify, is Thunderbird freezing on startup only when gdata is active?

And does this also happen with WIndows started in safe mode (and no data on dropbox)?
 https://support.microsoft.com/en-us/help/12376/windows-10-start-your-pc-in-safe-mode

**Reply by Tchibeck** [reply]
Ok for Windows Pro vs Home wich makes no difference.
I'll keep that in mind.

For data on Dropbox, i use it since Dropbox exists, more than a decade, so it's maybe not recommended  and i'll keep that in mind though i use it without any problem with all my other or past  computers Linux & Windows Home. 

Even though it's probably not the cause of the problem.

Now, for clarification. Only Thunderbird is freezing at startup, now i've moved 80% of my Dropbox files on non-synch repertory.

Next : when i start Thunderbird in safe mode : evrything's OK
When i start Thunderbird normaly, i have to wait 30 seconds, then it semmes to work normaly, beeing almost empty of local files.

But it still have bugs now and then when emptying recycle bin.

I have only one plugin : Gdata provider.

- When i switch it to non-active, then restart Thunderbird, it doesn't freeze.
- When i uninstall Gdata and replace with GSuite in Tab, Thunderbird doesn't freeze.
- When i have only Gdata installed, and have been waiting 30 seconds, till freeze cease, everything seem to be OK.... BUT :

If i switch off Gdata plugin, an switch it on again immediatly, then Thunderbird freezes for 30 seconds again.

Thank you for trying to help me..... It doesn't really matter as i can replace Gdata by Gsuite in TAB, but i would like to understand why Gdata plugin makes Thunderbird freeze.

And why it happens only on this computer (under W10 Pro) without having had any trouble before Thunderbird switch from releases 91.x.x to release 101.x.x and following.

**Comment by wsmwk** [comment]
You can get a performance profile https://support.mozilla.org/en-US/kb/profiling-thunderbird-performance
IIRC there are instructions there for the method needed to do it on startup.

**Reply by Tchibeck** [reply]
It' very kind of you... but 👍 

I uninstalled totally Thunderbird and any trace of it, even in the register... and believe me, since i bought my first comupter un 1982, when i say, "i cleaned any trace of it", that means any !

Profile, adress book, and so on ...were not on dropbox and are different from a computer to another.

I reinstalled a Thunderbird 102.x.x from scratch.
And the problem remain the same, as the Led Zep song.. ^^

I've been geeking for 40 years on computers beside my job wich had nothing to do with computers.

Now i'm retired, i 'm bored with softwares acting like viruses.
Bugs, bugs and bugs.

I apologize about developpers : it's pretty unfair.
But now i'm closer to my passing than to my birth, i haven't anymore time to spent debugging buggy softwares.

I wish i would have been a monk; not a slave of screens, computers, and internet.

So, if Gtab is buggy, i'll do without it. 
If Thunderbird is buggy, it will die slowly.

But i'll leave before.
Won't waste that precious time of life left anymore.

**Reply by wsmwk** [reply]
Great song, one of my first album purchases.

FWIW, at least for Thunderbird, uninstall+reinstall, registry cleans, etc essentially never help, except when AV is involved (which in all most all cases the AV is the proximate cause).

If you change your mind, we could use the data.
